examples/undocumented/python/evaluation_cross_validation_multiclass_storage.py

```diff
@@ -67,20 +67,20 @@ def evaluation_cross_validation_multiclass_storage (traindat=traindat, label_tra
         evaluation_criterion=evaluation_criterium, autolock=False, num_runs=3)
 
     # append cross validation parameter observer
-    multiclass_storage=ParameterObserverCV()
+    multiclass_storage=parameter_observer("ParameterObserverCV")
     cross_validation.subscribe_to_parameters(multiclass_storage)
 
     # perform cross-validation
     result=cross_validation.evaluate()
 
     # get first observation and first fold
-    obs = multiclass_storage.get_observations()[0]
-    fold = obs.get_folds_results()[0]
+    obs = multiclass_storage.get_observation(0).get("cross_validation_run")
+    fold = obs.get("folds", 0)
 
     # get fold ROC for first class
     eval_ROC = ROCEvaluation()
-    pred_lab_binary = MulticlassLabels.obtain_from_generic(fold.get_test_result()).get_binary_for_class(0)
-    true_lab_binary = MulticlassLabels.obtain_from_generic(fold.get_test_true_result()).get_binary_for_class(0)
+    pred_lab_binary = MulticlassLabels.obtain_from_generic(fold.get("test_result")).get_binary_for_class(0)
```

Comment:
These labels are a bit confusing don't you think?
Maybe we can call them "training" "validation" and "ground_truth" ?

Reply:
I would prefer something like:
-  test_result --> predicted_labels;
-  test_true_result --> ground_truth_labels;

Reply:
predicted, ground_truth? :)
In my opinion, the label should be short and details (such as that this is labels) should be in the type and the description.But if you dont like it go for it